.github/workflows/updatecli.yaml

```diff
@@ -18,6 +18,7 @@ jobs:
       - name: Set up Go
         uses: actions/setup-go@v4
         with:
+          cache: false
           go-version: 1.20.1
```

Comment:
Unrelated to the PR: it looks like the go version is not updated in this file

Reply:
Yep I noticed, but I would like first to understand how to free some space